core/src/main/java/org/elasticsearch/Version.java

```diff
@@ -296,7 +280,7 @@ public Version minimumCompatibilityVersion() {
         final int bwcMinor;
         if (this.onOrAfter(Version.V_6_0_0_alpha1_UNRELEASED)) {
             bwcMajor = major - 1;
-            bwcMinor = 0; // TODO we have to move this to the latest released minor of the last major but for now we just keep
+            bwcMinor = 4;
```

Comment:
Should we set this to 5 (for 5.5.0 now) and also add enforcement that it's increased any time a new minor version is bumped (e.g., to 6 if the version in 5.x is bumped)?